test/1_pool.js

```diff
@@ -43,20 +44,19 @@ function constructor(accounts) {
     it("constructor events", async () => {
         //SPARTANPROTOCOLv2
         base = await BASE.new() // deploy base
+        reserve = await RESERVE.new(base.address) // deploy base
         wbnb = await WBNB.new() // deploy wBNB
         Dao = await DAO.new(base.address)     // deploy daoV2
         router = await ROUTER.new(base.address, wbnb.address, Dao.address) //deploy router
         utils = await UTILS.new(base.address, router.address, Dao.address) // deploy utilsV2
         poolFactory = await POOLFACTORY.new(base.address,  wbnb.address, Dao.address) 
         token1 = await TOKEN.new()     
-        await Dao.setGenesisAddresses(router.address, utils.address, utils.address, utils.address, utils.address,poolFactory.address, utils.address);
-    
-       
+        await Dao.setGenesisAddresses(router.address, utils.address, utils.address, utils.address, utils.address,poolFactory.address, utils.address, reserve.address);
     
         await base.transfer(acc1, _.getBN(_.BN2Str(100000 * _.one)))
         await base.transfer(acc2, _.getBN(_.BN2Str(100000 * _.one)))
         await base.transfer(acc0, _.getBN(_.BN2Str(100000 * _.one)))
-        await base.transfer(router.address, _.getBN(_.BN2Str(100000 * _.one)))
+        await base.transfer(reserve.address, _.getBN(_.BN2Str(100000 * _.one)))
```

Comment:
Change / add automated tests for Pool to handle SPARTA being emitted to the reserve contract